app/controllers/api/v1/media_controller.rb

```diff
@@ -31,7 +31,7 @@ def status_code_for_media_attachment
   end
 
   def set_media_attachment
-    @media_attachment = current_account.media_attachments.unattached.find(params[:id])
+    @media_attachment = current_account.media_attachments.find(params[:id])
```

Comment:
This allows updating media attachments that have already been attached, but nothing in this controller is responsible for attaching media attachments to statuses. I see no other way to allow editing media attachments during status editing.

Reply:
Ok, you're right, but it would allow the media file to be edited/replaced after the fact without the status being marked as modified, though. Not sure how to handle that.

Reply:
I have made it so you can't update `:file` (as was intended originally). I don't have a simple answer otherwise. The updates to description and so on won't federate unless a status update is made. One potential solution is to only allow updating media attachments as nested resources through the status update API but it seems kind of ugly especially in terms of how to modify the web UI for that (media edit modal relying on being able to update individual attachments instantly)

Reply:
Well, that's for authoring edited toots. I wonder if it wouldn't be best to put that to another PR, as I think handling incoming edited toots is worthwhile enough and deserving of its own PR.